.evergreen/run-socks5-tests.sh

```diff
@@ -51,31 +39,33 @@ if [ "Windows_NT" = "$OS" ]; then
   SOCKS5_SERVER_SCRIPT=$(cygpath -w "$SOCKS5_SERVER_SCRIPT")
 fi
 
-# First, test with Socks5 + authentication required
-"$PYTHON_BINARY" "$SOCKS5_SERVER_SCRIPT" --port 1080 --auth username:p4ssw0rd --map "127.0.0.1:12345 to $FIRST_HOST" &
-SOCKS5_PROXY_PID=$!
-if [[ $TEST_SOCKS5_CSFLE == "true" ]]; then
-  setup_fle
-  [ "$SSL" == "nossl" ] && [[ "$OSTYPE" == "linux-gnu"* ]] && \
-  env MONGODB_URI='mongodb://127.0.0.1:12345/?proxyHost=127.0.0.1&proxyUsername=username&proxyPassword=p4ssw0rd' \
-  npm run check:csfle
-else
-  env SOCKS5_CONFIG='["127.0.0.1",1080,"username","p4ssw0rd"]' npm run check:socks5
-fi
-kill $SOCKS5_PROXY_PID
-
-# Second, test with Socks5 + no authentication
-"$PYTHON_BINARY" "$SOCKS5_SERVER_SCRIPT" --port 1081 --map "127.0.0.1:12345 to $FIRST_HOST" &
-SOCKS5_PROXY_PID=$!
-if [[ $TEST_SOCKS5_CSFLE == "true" ]]; then
-  setup_fle
-  [ "$SSL" == "nossl" ] && [[ "$OSTYPE" == "linux-gnu"* ]] && \
-    env MONGODB_URI='mongodb://127.0.0.1:12345/?proxyHost=127.0.0.1&proxyPort=1081' \
+if [ "$AUTH" = "auth" ]; then 
```

Comment:
This looks like a big diff but it's actually only wrapping everything in this if statement, conditionally running one or the other suite depending on whether or not auth is enabled.